### ISSUE TITLE
Storing currency with a comma instead of a dot

### ISSUE THREAD

**Issue by ttomdewit** [vyuldashev/nova-money-field#5]
Hi there 👋 ,

First of all thank you for your work on this package!

Would it be at all possible to cater for the following?

In the Netherlands we type prices as "€29,50". When I store a new price with the current implementation I need to type "29.50". It'll translate correctly to "29,50", though.

Thank you for looking into this.

Kind regards,

Tom

**Comment by vyuldashev** [comment]
Hi. Any ideas how to implement this?

**Comment by ttomdewit** [comment]
Hi!

In the past I've cheated and used `str_replace` to get what I needed in the database since I've only supported one currency, so it was an easy decision. Unfortunately that won't work for this package.

Perhaps we could look at the user's locale, or allow them to tell the package what "separator" to use via a new method?

Kind regards,

Tom

**Comment by vyuldashev** [comment]
@ttomdewit I am not sure to do it on backend. On frontend we can use `toLocaleString` and it will to the trick. But, on the backend how to convert it back to float? :)

**Comment by ttomdewit** [comment]
@vyuldashev I think we could add two "separator"-methods that allows the user to tell what both separators should be, much like `number_format()` does with parameter 3 and 4. Perhaps that'll do the trick?

More info on `number_format` can be found on http://php.net/manual/en/function.number-format.php.

**Comment by vyuldashev** [comment]
@ttomdewit If number is "500 000 000.50" how to you type it?

**Comment by ttomdewit** [comment]
@vyuldashev So we're talking about five hundred million and 50 cents? That would be "500.000.000,50" over here whereas, for example in the US, it would translate to "500,000,000.50", IIRC.

**Comment by ttomdewit** [comment]
@vyuldashev No, I'd like to put "500.000.000,50". We use the `,` to separate decimals and we use the `.` to separate thousands.
